libs/domains/variables/feature/src/lib/variable-list/variable-list.tsx

```diff
@@ -517,6 +527,14 @@ export function VariableList({
                   ) : (
                     flexRender(header.column.columnDef.header, header.getContext())
                   )}
+                  {i === 1 && (
+                    <span className="absolute right-[-36px] top-[8px]">
```

Comment:
```suggestion
                    <span className="absolute -right-9 top-[7px]">
```

(7px is necessary to be center in the parent)